src/site/stages/build/process-cms-exports/transform.js

```diff
@@ -1,10 +1,23 @@
+/* eslint-disable import/no-dynamic-require */
+
+const fs = require('fs');
+const path = require('path');
+
 const { mapKeys, camelCase } = require('lodash');
 const { getContentModelType } = require('./helpers');
-const pageTransform = require('./transformers/page');
 
-const transformers = {
-  'node-page': pageTransform,
-};
+// Dynamically read in all the transformers
+// They must be named after the content model type (E.g. node-page.js)
+const transformersDir = path.join(__dirname, 'transformers');
+const transformers = fs
+  .readdirSync(transformersDir)
+  .filter(name => name.endsWith('.js'))
+  .reduce((t, fileName) => {
+    const contentModelType = fileName.slice(0, -3); // Take of the '.js'
```

Comment:
This is sweet!

Small suggestion - I  personally prefer `path` here to help add a bit of meaning to the string manipulation.  

`path.extname(fileName) === '.js'` -> doesn't do much in this case, but useful if you're checking for multiple file extensions
 `path.parse(fileName).name` -> removes file extension

Reply:
Ooh, cool! Didn't know about `parse().name`. Good call :+1: